src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -229,11 +236,20 @@
     if (!isContainUrl(item?.user?.image) || item?.user?.name === ANONYMOUS_USER) {
       return (
         <View style={styles.mr7}>
-          <AnonymousIcon
-            color={anon_user_info_color_code}
-            emojiCode={anon_user_info_emoji_code}
-            size={normalize(48)}
-          />
+          {betterSocialMember &&
+          item.user_id === betterSocialMember[memberChat?.user_id]?.user_id ? (
+            <AnonymousIcon
+              color={betterSocialMember[memberChat?.user_id]?.anon_user_info_color_code}
+              emojiCode={betterSocialMember[memberChat?.user_id]?.anon_user_info_emoji_code}
+              size={normalize(48)}
+            />
+          ) : (
+            <AnonymousIcon
+              color={anon_user_info_color_code}
+              emojiCode={anon_user_info_emoji_code}
+              size={normalize(48)}
+            />
+          )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This block of code checks if the current user is the same as the better social member, and if so, displays an anonymous icon with specific color and emoji code. If not, it displays a default anonymous icon. This logic seems correct as it provides a visual distinction between different types of users.
```diff
+           {betterSocialMember &&
+           item.user_id === betterSocialMember[memberChat?.user_id]?.user_id ? (
+             <AnonymousIcon
+               color={betterSocialMember[memberChat?.user_id]?.anon_user_info_color_code}
+               emojiCode={betterSocialMember[memberChat?.user_id]?.anon_user_info_emoji_code}
+               size={normalize(48)}
+             />
+           ) : (
+             <AnonymousIcon
+               color={anon_user_info_color_code}
+               emojiCode={anon_user_info_emoji_code}
+               size={normalize(48)}
+             />
+           )}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->